index.js

```diff
@@ -194,3 +194,27 @@ form.addEventListener('submit', (e) => {
 email.addEventListener('input', () => {
   email.setCustomValidity('');
 });
+
+// save data to local storage
+const name = document.querySelector('.name');
+const message = document.querySelector('textarea');
+
+const getDataFromStorage = () => {
+  const storageData = JSON.parse(localStorage.getItem('data'));
+  if (storageData) {
+    name.value = storageData.name;
+    email.value = storageData.email;
+    message.value = storageData.message;
+  }
+};
+
+const saveDataToStorage = () => {
+  const data = {
+    name: name.value,
+    email: email.value,
+    message: message.value,
+  };
+  localStorage.setItem('data', JSON.stringify(data));
+};
+window.addEventListener('load', getDataFromStorage);
+form.addEventListener('submit', saveDataToStorage);
```

Comment:
Great work so far! As specified in the project requirements, user input should be `stored` in the `localStorage` whenever they input or change data in the input field. Your current code works well, but it only stores the user's data in the `localStorage` when they hit the `submit` button. However, this is not the intended behavior. Please modify your code to store the user's data in the `localStorage` whenever they input or change data in the input field. Thank you!
<img width="873" alt="Screenshot 2023-04-15 at 08 43 52" src="https://user-images.githubusercontent.com/107192076/232201787-214288b4-e4c5-4171-b4be-fec6cec8184d.png">

Reply:
Thank you for the review